qiita_db/sql_connection.py

```diff
@@ -460,176 +450,199 @@ def execute_fetchall(self, sql, sql_args=None):
 
         return result
 
-    def _check_queue_exists(self, queue_name):
-        """Checks if queue `queue_name` exists in the handler
 
-        Parameters
-        ----------
-        queue_name : str
-            The name of the queue
+class Transaction(object):
```

Comment:
I've just added `Transaction` to the classes list and removed the examples. I will add the examples back once I'm 100% sure that I don't have to do any further change in the Transaction class (maybe adding some functionality that is needed and I didn't foresee). In order to not forget about this, I've added an item in #1262